packages/at_onboarding_cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.5.1
```

Comment:
Should be 1.6.0 as this is definitely new functionality rather than a patch.

Reply:
@cpswan thank you yes

Reply:
have made that change